sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -28,9 +24,15 @@ import { AppSupportingUsersModule } from "./app.supporting-users.module";
 import { ZeebeModule } from "@sims/services";
 import { LoggerModule } from "@sims/utilities/logger";
 import { ConfigModule } from "@sims/utilities/config";
+import { DatabaseModule } from "@sims/sims-db";
 import { NotificationsModule } from "@sims/services/notifications";
+import {
```

Comment:
None of these modules are supposed to be needed on the API. Can you please review and remove them all?
Same for the other individual API modules.